src/commands/General/qna.ts

```diff
@@ -58,7 +58,7 @@ export default class QnaCommand {
 
     @Slash({
         name: 'solve',
-        description: 'Resolve this QNA topic',
+        description: 'Giải quyết vấn đề này',
```

Comment:
```suggestion
        description: 'Giải quyết xong vấn đề',
```

hoặc cái gì đó mang nghĩa tương tự